src/CommandQueue.ts

```diff
@@ -47,6 +56,20 @@ export class CommandQueue {
         registerDomEvents: (payload: any): void => {
             this.orchestration.registerDomEvents(payload);
         },
+        recordEvent: (payload: RecordEventInput) => {
+            const type = payload.event_type;
+            const eventData = payload.event_data;
+            if (
+                type === undefined ||
+                eventData === undefined ||
+                typeof type !== 'string' ||
+                typeof eventData !== 'object'
+            ) {
+                throw new Error('IncorrectParametersException: invalid input.');
```

Comment:
nit: For consistency.
```suggestion
                throw new Error('IncorrectParametersException')
```